diagnostics/TUTT/TUTT_calc_model.py

```diff
@@ -0,0 +1,648 @@
+#Jun 29 2021
```

Comment:
This file and TUTT_clac_obs.py (TUTT_calc_obs.py?) are nearly identical. I recommend  combining them into a single file (e.g., TUTT_calc.py) that loops through an array or dictionary of input files in model and obs input directories